### DISCUSSION TITLE
Dependency update?

### DISCUSSION THREAD

**Discussion by erenirmak** [pymc-devs/pymc3#7029] [Q&A] [answered]
Some of the dependencies have newer versions, like numpy, scipy. Will PyMC3 be updated for the compatibility with the newer versions?

**Comment by ricardoV94** [comment, marked as answer]
We are not planning to do any further maintenance on the PyMC3 project.

**Reply by twiecki** [reply]
The new version is just called PyMC, currently at version 5.9.0 which has up-to-date dependencies.

**Reply by erenirmak** [reply]
oh..thank you
